styleguide/_themes/global/scss/bootstrap_overrides.scss

```diff
@@ -80,10 +79,33 @@
 
     &:focus,
     &:hover {
-      background-color: $gray-darker;
+      background-color: darken($brand-primary, 5%);
     }
   }
-  .navbar-nav .navbar-topLink-dropDown.navbar-topLink:hover {
-    background-color: $gray-dark;
+
+  .navbar-dropDownMenu > li.navbar-dropDownItem {
+    background-color: $brand-primary;
   }
+
+  .navbar-dropDownMenu > li.navbar-dropDownItem > .navbar-dropDown-triggerActive {
+    background: darken($brand-primary, 5%);
+  }
+
+}
+
+.navbar-search.navbar-search-open,
+.no-js .navbar-search {
+  width: 20%;
+}
+
+.navbar-search-open > .navbar-icon-search,
+.no-js > .navbar-search > .navbar-icon-search {
+  background: $white;
+  color: $gray-darker;
+  z-index: 11;
+}
+
+.navbar-search-open .navbar-search-submit,
+.no-js > .navbar-search > .navbar-search-submit {
```

Comment:
The bootstrap overrides are getting a bit ridiculous.  Not your fault I realize, but we need to discuss removing the offending code from bootstrap altogether.